src/components/schema-editor/schema-editor.component.tsx

```diff
@@ -40,8 +40,12 @@ const SchemaEditor: React.FC<SchemaEditorProps> = ({
 
   const inputDummySchema = useCallback(() => {
     const dummySchema: OHRIFormSchema = {
-      encounterType: "",
+      encounterType: "Test",
       name: "Sample Form",
+      processor: "EncounterFormProcessor",
+      referencedForms: [],
+      uuid: "f7a2a8f3-2ba9-49fb-a0d8-aaab0edbc1f0",
```

Comment:
Was this change needed?

Reply:
In hindsight, no. Thanks for catching this.